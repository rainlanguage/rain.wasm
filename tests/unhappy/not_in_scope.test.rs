#[macro_use]
extern crate wasm_bindgen_utils;

impl_wasm_traits!(SomeType);

fn main() {}
