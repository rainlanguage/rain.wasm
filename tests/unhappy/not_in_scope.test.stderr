error[E0412]: cannot find type `SomeType` in this scope
 --> tests/unhappy/not_in_scope.test.rs:4:19
  |
4 | impl_wasm_traits!(SomeType);
  |                   ^^^^^^^^ not found in this scope
